python/ray/autoscaler/autoscaler.py

```diff
@@ -824,3 +854,22 @@ def add_hash_of_file(fpath):
         _hash_cache[conf_str] = hasher.hexdigest()
 
     return _hash_cache[conf_str]
+
+def request_cores(redis_address, cores, redis_password=None):
+    """Remotely request some CPU cores from the autoscaler.
+    This function is to be called e.g. on a node before submitting a bunch of
```

Comment:
newline here

Reply:
I've made the formatting changes requested, although I wasn't able to run the format script on my machine for some reason so some things may still be off.